projects/media-viewer/src/lib/bookmark/bookmark.module.ts

```diff
@@ -0,0 +1,26 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { HttpClientModule } from '@angular/common/http';
+import { BookmarkComponent } from './components/bookmark.component';
+import { RouterModule } from '@angular/router';
+import { BookmarkApiService } from './services/bookmark-api.service';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    HttpClientModule,
+    RouterModule
+  ],
+  declarations: [
+    BookmarkComponent,
+  ],
+  entryComponents: [
+  ],
+  providers: [
+    BookmarkApiService
+  ],
+  exports: [
+    BookmarkComponent
+  ]
+})
+export class BookmarkModule { }
```

Comment:
do u really need a module? don't think is a module

Reply:
Honestly, I only made it a module as I was basing my work from what had been done with redactions and that had a todo for making it into a module...https://github.com/hmcts/media-viewer/blob/master/projects/media-viewer/src/lib/media-viewer.module.ts#L64
Happy to conform to how redactions are used currently if you prefer?

Reply:
yeah, I put that in there.... I think we should have a conversation about how to structure the app - atm we have no rules. leave it as it is